src/main/java/ch/ivyteam/ivy/maven/engine/Slf4jSimpleEngineProperties.java

```diff
@@ -23,79 +23,72 @@
 
 /**
  * Sets the logging properties for the ivy engine.
- * 
+ *
  * @author Reguel Wermelinger
  * @since 6.0.0
  */
-public class Slf4jSimpleEngineProperties
-{
+public class Slf4jSimpleEngineProperties {
+
   private static final String DEFAULT_LOG_LEVEL = SimpleLogger.DEFAULT_LOG_LEVEL_KEY;
   private static final List<String> INTERESTING_LOGGERS = Arrays.asList(
-          "ch.ivyteam.ivy.scripting.dataclass.internal.InMemoryEngineController", // engine start/stop
+          "ch.ivyteam.ivy.scripting.dataclass.internal.InMemoryEngineController", // legacy > engine start/stop: (up to 9.3)
+          "ch.ivyteam.ivy.server.build.InMemoryEngineController", // engine start/stop

```

Comment:
that's my real change: move an important build infrastructure class to a new bundle+namespace